### PR TITLE
Edge cie 5005 first login password reset

### PR DESCRIPTION
**Pull request by TzviLipschitz**
release/10.17
https://github.com/SoftwareAG/c8y-docs/pull/1913

release/10.18
https://github.com/SoftwareAG/c8y-docs/pull/1913

release/2024
PR (merge when 2024 edge on k8s is about to be released)

Develop
PR (merge when 2024 edge on k8s is about to be released)

### CODE REVIEW

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 1]
```diff
@@ -92,7 +97,7 @@ After a successful deployment, you must access both the {{< management-tenant >}
 
 If you are logging in for the first time, you will see a cookie banner at the bottom:
 
-![Login prompt](/images/users-guide/getting-started/getting-started-cookie-banner.png)
+![Login prompt](/images/edge-k8s/edge-k8s-getting-started-cookie-banner.png)
```
[the remark this answers is not in the document]
Comment (TzviLipschitz) [reply content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md]: Is this still relevant? The screenshot was present before this PR but has been split in to two images

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 2]
```diff
@@ -80,22 +76,42 @@ Enter one of the following URLs in the browser:
 * `https://<domain_name>`
 * `https://management.<domain_name>`
```
Comment (bbyreddy) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:77]: - [ ] Add text to indicate that `https://<domain_name>` should be used to log into the edge tenant and `https://management.<domain_name>` to log into the management tenant.
- [ ] This statement will replace the lines removed
removed line#89:  To log in to the {{< management-tenant >}}, use the URL `https://management.<domain_name>`.
removed line#91:  To log in to the {{< product-c8y-iot >}}  Edge tenant, use the URL `https://edge.<domain_name>`.

File: content/edge/operating-edge-bundle/health-check.md [hunk 3]
```diff
@@ -14,7 +14,9 @@ The following instructions show how to check the network connectivity of the pla
 
 The following commands will show the interface and network settings of the machine:
 
-	[admin@iot-edge-server ~]$ ip a
+```shell
+	ip a
```
Comment (bbyreddy) [inline content/edge/operating-edge-bundle/health-check.md:18]: - [ ] Do we need to remove the preceeding spaces?

**Review by bbyreddy — COMMENTED** [review]
@TzviLipschitz , please find my comments.

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md (hunk 2 above)
Comment (TzviLipschitz) [reply content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md]: Lines 89 and 91 have moved to line 105 and 107 respectively. I have added the additional text

File: content/edge/operating-edge-bundle/health-check.md (hunk 3 above)
Comment (TzviLipschitz) [reply content/edge/operating-edge-bundle/health-check.md]: Removed the space

File: content/edge/edge-installation-bundle/installing-edge-using-gui.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ While configuring the network on VMware based hypervisors, do not us the IP addr
 Do not use the root credentials to perform any task. The root credentials must be used only when asked by {{< company-sag >}} support personnel. Using it otherwise might void the appliance support.
     {{< /c8y-admon-important >}}
 
-8. Create an administrator account to access the {{< product-c8y-iot >}} Edge tenant and the {{< product-c8y-iot >}} Edge {{< management-tenant >}}, and click **Next**.
+8. Create an administrator account to access the Edge tenant and the {{< management-tenant >}}, and click **Next**.
```
Comment (MWindel) [inline content/edge/edge-installation-bundle/installing-edge-using-gui.md:51]: @TzviLipschitz Why did you remove the Cumulocity IoT reference here?

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 5]
```diff
@@ -68,34 +64,54 @@ On Windows machines, add the same entry to *C:\Windows\System32\drivers\etc\host
...
 
-If you are logging in for the first time, you will see a cookie banner at the bottom:
+If you are logging in for the first time, you will see a cookie banner at the bottom of the login screen:
 
-![Login prompt](/images/users-guide/getting-started/getting-started-cookie-banner.png)
+![Cookie Banner](/images/edge-k8s/edge-k8s-cookie-banner.png)
 
 {{< c8y-admon-info >}}
-The cookie banner is turned on by default on the {{< product-c8y-iot >}} Edge instances. This feature can be configured, see [{{< enterprise-tenant >}} > Customizing your platform > Branding](/enterprise-tenant/customization/#branding).
+The cookie banner is turned on by default. This feature can be configured, see [{{< enterprise-tenant >}} > Customizing your platform > Branding](/enterprise-tenant/customization/#branding).
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:114]:
```suggestion
The cookie banner is turned on by default. This feature can be configured. For more information see [{{< enterprise-tenant >}} > Customizing your platform > Branding](/enterprise-tenant/customization/#branding).
```

File: content/edge/edge-installation-bundle/installing-edge-using-gui.md (hunk 4 above)
Comment (TzviLipschitz) [reply content/edge/edge-installation-bundle/installing-edge-using-gui.md:51]: @bbyreddy Can you provide more context here?

Comment (bbyreddy) [reply content/edge/edge-installation-bundle/installing-edge-using-gui.md:51]: @MWindel , all these occurances are not referring to the Cumulocity IoT Edge product, rather they are referring to the tenant named "edge" similar to the tenant named "management". @TzviLipschitz , maybe we can rename "Edge" to "edge", similar to the way management tenant is referred.

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 6]
```diff
@@ -68,34 +64,54 @@ On Windows machines, add the same entry to *C:\Windows\System32\drivers\etc\host
...
-* `https://<domain_name>`
-* `https://management.<domain_name>`
+* To log in to the edge tenant: `https://<domain_name>`
+* To log in to the management tenant: `https://management.<domain_name>`
+
+The login screen appears.
+
+
+{{< c8y-admon-important >}}
+{{< product-c8y-iot >}} Edge is installed with the admin user **"admin"** and password **"admin-pass"**.
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:83]:
```suggestion
{{< product-c8y-iot >}} Edge is installed with the admin user "admin" and password "admin-pass".
```

Comment (MWindel) [reply content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:83]: @TzviLipschitz Please do not use double formatting (here, bold and quotation marks). For content such as login credentials, we use quotation marks.

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 7]
```diff
@@ -68,34 +64,54 @@ On Windows machines, add the same entry to *C:\Windows\System32\drivers\etc\host
...
+{{< /c8y-admon-important >}}
+
+![Login prompt](/images/edge-k8s/edge-k8s-login-banner.png)
+
+Change the password on first login.
+
+{{< c8y-admon-important >}}
+The Email address to change the password on the first login is the one you specified in the Edge CR (or myown@iot.com if you followed the Quickstart installation steps).
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:91]:
```suggestion
The email address to change the password on the first login is the one you specified in the {{< product-c8y-iot >}} Edge CR (or myown@iot.com if you followed the Quickstart installation steps).
```

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 8]
```diff
@@ -68,34 +64,54 @@ On Windows machines, add the same entry to *C:\Windows\System32\drivers\etc\host
...
+Alternatively you can also retrieve the Email address by running the below command:
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:92]:
```suggestion
Alternatively, run the following command to retrieve the email address:
```

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 9]
```diff
@@ -68,34 +64,54 @@ On Windows machines, add the same entry to *C:\Windows\System32\drivers\etc\host
...
+
+`kubectl get edge c8yedge -n c8yedge -o jsonpath='{.spec.email}' && echo`
+
+_Substitute the Edge name and namespace name, which is currently **c8yedge** in the command, with the specific Edge name and namespace name you have specified in your Edge CR._
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:96]:
```suggestion
Substitute the {{< product-c8y-iot >}} Edge name and namespace name, which is currently **c8yedge** in the command, with the specific {{< product-c8y-iot >}} Edge name and namespace name you have specified in your {{< product-c8y-iot >}} Edge CR.
```

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 10]
```diff
@@ -68,34 +64,54 @@ On Windows machines, add the same entry to *C:\Windows\System32\drivers\etc\host
...
+{{< /c8y-admon-important >}}
 
-The login screen appears. If this is your first login, log in with user "admin" and password "admin-pass" and change the password.
+![Reset password](/images/edge-k8s/edge-k8s-reset-password.png)
 
 {{< c8y-admon-important >}}
-After a successful deployment, you must access both the {{< management-tenant >}} and {{< product-c8y-iot >}}  Edge tenants and change the admin credentials.
+You must access both the management and edge tenants and change the respective admin credentials.
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:102]: @TzviLipschitz We do not have different tenants for Edge and Cumulocity IoT. Please leave the former tenant shortcodes.

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 11]
```diff
@@ -68,34 +64,54 @@ On Windows machines, add the same entry to *C:\Windows\System32\drivers\etc\host
...
 {{< /c8y-admon-important >}}
 
-- To log in to the {{< management-tenant >}}, use the URL `https://management.<domain_name>`.
+- To log in to the edge tenant, use the URL `https://<domain_name>`.
 
-- To log in to the {{< product-c8y-iot >}}  Edge tenant, use the URL `https://edge.<domain_name>`.
+- To log in to the management tenant, use the URL `https://management.<domain_name>`.
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:107]:
```suggestion
- To log in to the {{< management-tenant >}}, use the URL `https://management.<domain_name>`.
```

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 12]
```diff
@@ -118,7 +134,7 @@ Finally, click **Login** to enter {{< product-c8y-iot >}} Edge. Initially, you w
 To explicitly log out, click the **User** button at the right of the top bar, then select **Logout** from the context menu.
 
 {{< c8y-admon-info >}}
-The maximum number of failed logins (due to invalid credentials), after which a user is locked, can be configured by the {{< management-tenant >}} on platform level, see *{{< product-c8y-iot >}} Core - Operations guide*. The default value is 100.
+The maximum number of failed logins (due to invalid credentials), after which a user is locked, can be configured by the management tenant on platform level, see *{{< product-c8y-iot >}} Core - Operations guide*. The default value is 100.
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:137]: @TzviLipschitz Please use the tenant shortcodes as before ({{< management-tenant >}} ).

File: content/edge/edge-installation-bundle/installing-edge-using-gui.md (hunk 4 above)
Comment (MWindel) [reply content/edge/edge-installation-bundle/installing-edge-using-gui.md:51]: @bbyreddy "management tenant"/"{{< management-tenant >}} " is not a name but a tenant type. So which tenant type would the "edge tenant" be?

**Review by bbyreddy — APPROVED** [review]

**Review by bbyreddy — APPROVED** [review]

**Review by MWindel — APPROVED** [review]
